<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="stepSpinBox1">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>110</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepSpinBox2">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>110</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepSpinBox3">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>110</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="directionComboBox1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>horaire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>anti-horaire</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="directionComboBox2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>horaire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>anti-horaire</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="directionComboBox3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>horaire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>anti-horaire</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>DÃ©marrer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
